front/src/components/Articles.jsx

```diff
@@ -128,7 +128,7 @@ const ConnectedArticles = (props) => {
 
   return (
     <section className={styles.section}>
-      <h1>Articles for {displayName}</h1>
+      <h1>Articles for {displayName} </h1>
```

Comment:
Ce changement n'est pas nécessaire : 

```suggestion
      <h1>Articles for {displayName}</h1>
```

Reply:
ok